hypothesis-python/tests/ghostwriter/test_ghostwriter_cli.py

```diff
@@ -41,8 +44,11 @@
             "--roundtrip json.loads json.dumps --except ValueError",
             lambda: roundtrip(json.loads, json.dumps, except_=ValueError),
         ),
+        ("hypothesis.strategies.builds", lambda: fuzz(hypothesis.strategies.builds)),
+        # Discover methods in class
+        ("hypothesis.errors.StopTest", lambda: magic(StopTest)),
         # Imports submodule (importlib.import_module passes; __import__ fails)
-        ("hypothesis.errors.StopTest", lambda: fuzz(StopTest)),
+        ("hypothesis.strategies", lambda: magic(hypothesis.strategies)),
```

Comment:
What are each of these new tests checking for?  Is that comment about import semantics still correct, even though we've changed what we're testing?

I'd prefer to just swap `magic(StopTest)` for `fuzz(StopTest)`, but if we're adding more we also need to add corresponding comments.

Reply:
Because there's no classmethod and staticmethod in StopTest so I swap it out, I can add it back in cause it was generating nothing before (remember I remove the class being tested itself?)

Reply:
> What are each of these new tests checking for?

I think the comment above kinda indicates something? for the `magic(hypothesis.strategies)` it is a submodule and for `magic(StopTest)` it discovers the methods in class (though I think it's a bad example as there's nothing in StopTest I believe)